src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -1887,6 +1887,58 @@ public sealed partial class ZoneMembershipCondition : System.Security.ISecurityE
         public System.Security.SecurityElement ToXml(System.Security.Policy.PolicyLevel level) { throw null; }
     }
 }
+namespace System.ServiceProcess.ServiceController
+{
+    public sealed partial class ServiceControllerPermission : System.Security.Permissions.ResourcePermissionBase
```

Comment:
Re namespace, they should match desktop so `System.ServiceProcess`. Again you can use https://apisof.net or ILSPY to check. So you need to fix here.